src/ResourceManager/Compute/Commands.Compute/help/Remove-AzureRmVMDiagnosticsExtension.md

```diff
@@ -25,7 +25,7 @@ You must pass the output of this cmdlet to the Update-AzureRmVM cmdlet to implem
 
 ### Example 1: Remove the Diagnostics extension from a virtual machine
 ```
-PS C:\>Remove-AzureRmVMDiagnosticsExtension -ResourceGroupName "ResourceGroup11" -VMName "ContosoVM22" | Update-AzureRmVM
```

Comment:
@yantang-msft the description below says:

_The command passes the result to the Update-AzureRmVM cmdlet by using the pipeline operator. That command updates the virtual machine._

Any reason why you want to remove the piping to the `Update-AzureRmVM` cmdlet?